magicbyirineu/Modules/CardList/View/RoundedView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RoundedView.swift
+//  magicbyirineu
+//
+//  Created by andre.antonio.filho on 27/02/19.
+//  Copyright © 2019 DanielLima. All rights reserved.
+//
+
+import UIKit
+
+class RoundedView: UIView{
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        self.layer.cornerRadius = 5
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func setupAsShadowView(){
```

Comment:
Esse método pode ser private. E ser chamado no init da view. 

```suggestion
   private func setupView(){ 
          setupRoundedView()          
          if type == .roundedAndShaded {
               setupShadow()
          }
```

Reply:
e ai vc pode configurar a view de acordo com o tipo. Fica um pouco mais elegante. Mas o modo que vc fez não está errado